src/config/syscheck-config.c

```diff
@@ -100,7 +100,7 @@ int initialize_syscheck_configuration(syscheck_config *syscheck) {
     syscheck->wdata.fd                        = NULL;
 #endif
 #ifdef WIN32
-    syscheck->value_limit                     = 100000;
+    syscheck->reg_entry_limit                     = 100000;
```

Comment:
Tabulation here is not aligned with the other vars

Reply:
done